Dictionary.java

```diff
@@ -8,33 +8,244 @@
 
 public class Dictionary {
 
+    // Config
+    private static final String DICT_FILE = "dictionary.txt";
+
     // Props
     private List<String> fullWordList;
     private List<String> filteredList;
-    private Map<Character, Float> letterFrequency = new HashMap<>();
+    private List<FrequencyEntry> letterFrequency;
 
     // Init
     public Dictionary() {
+        this.init();
+    }
+
```

Comment:
Update javadoc for initializers. Also could probably remove the `init()` functions in favor of just using the constructors.